### PR TITLE
feat(deps): update wish and keygen

### PR DESCRIPTION
**Pull request by caarlos0**
update wish and keygen deps

### CODE REVIEW

File: client_auth.go [hunk 1]
```diff
@@ -149,28 +149,26 @@ func tryNewKey() (gossh.AuthMethod, error) {
...
 		"key.type", signer.PublicKey().Type(),
 		"key.fingerprint", gossh.FingerprintSHA256(signer.PublicKey()),
 	)
 
-	if key.KeyPairExists() {
-		return gossh.PublicKeys(signer), nil
+	if !key.KeyPairExists() {
+		if err := key.WriteKeys(); err != nil {
+			return nil, fmt.Errorf("could not write key: %w", err)
+		}
```
Comment (aymanbagabas) [inline client_auth.go:168]: You don't need this since you're using `WithWrite` option

File: cmd/wishlist/main.go [hunk 2]
```diff
@@ -133,7 +133,7 @@ var serverCmd = &cobra.Command{
 			}()
 		}
 
-		k, err := keygen.New(".wishlist/server", nil, keygen.Ed25519)
+		k, err := keygen.New(".wishlist/server", keygen.WithKeyType(keygen.Ed25519))
```
Comment (aymanbagabas) [inline cmd/wishlist/main.go:136]:
```suggestion
		k, err := keygen.New(".wishlist/server_ed25519", keygen.WithKeyType(keygen.Ed25519))
```

File: client_auth.go (hunk 1 above)
Comment (caarlos0) [reply client_auth.go:168]: ah, forgot to remove withwrite

File: client_auth.go [hunk 3]
```diff
@@ -149,28 +149,26 @@ func tryNewKey() (gossh.AuthMethod, error) {
 		return nil, fmt.Errorf("could not create client key: %w", err)
 	}
 
-	key, err := keygen.New(path, nil, keygen.Ed25519)
+	key, err := keygen.New(path, keygen.WithKeyType(keygen.Ed25519))
```
Comment (aymanbagabas) [inline client_auth.go:152]:
```suggestion
	key, err := keygen.New(path+"_ed25519", keygen.WithKeyType(keygen.Ed25519))
```

**Review by aymanbagabas — APPROVED** [review]

**Pull request merged**